modules/rke2/master/outputs.tf

```diff
@@ -9,8 +9,14 @@ output "kubeconfig" {
 }
 
 output "master_ips" {
-  value = join("," , aws_instance.master.*.public_ip,aws_instance.master2-ha.*.public_ip)
+  value = length(aws_eip.master_with_eip) > 0 ? join("," , aws_eip.master_with_eip.*.public_ip,aws_eip.master2_with_eip.*.public_ip): join("," , aws_instance.master.*.public_ip,aws_instance.master2-ha.*.public_ip)
```

Comment:
Suggestion: Can we simply use `var.create_eip` flag here instead of `length(aws_eip.master_with_eip) > 0`